crates/router/src/configs/settings.rs

```diff
@@ -96,6 +96,7 @@ pub struct Settings<S: SecretState> {
     pub payouts: Payouts,
     pub payout_method_filters: ConnectorFilters,
     pub applepay_decrypt_keys: SecretStateContainer<ApplePayDecryptConfig, S>,
+    pub paze_decrypt_keys: SecretStateContainer<PazeDecryptConfig, S>,
```

Comment:
Actually PAZE will offer universal keys which will be used by all the merchants hence this need not be optional because we will always have the Paze configs.

Reply:
Actually PAZE will offer universal keys which will be used by all the merchants hence this need not be optional because we will always have the Paze configs.

Reply:
But this may not be the case for those self-hosting Hyperswitch?

Reply:
Done. `paze_decrypt_keys` is now made an option